packages/yourloops/test/integration/mock/team.api.mock.ts

```diff
@@ -164,6 +165,46 @@ export const teamThree: Team = {
   }]
 }
 
+export const PatientNotification: ITeam = {
+  name: 'PatientNotification',
+  id: '63c7b7989cacc878ecce2c40',
+  code: '679517388',
+  type: TeamType.medical,
+  monitoring: {
+    ...monitoringParameters,
+    enabled: true
+  },
+  phone: '0478780000',
+  email: 'patientNotification@hey.hey',
+  address: {
+    line1: '6 rue des champs',
+    line2: '',
+    zip: '75000',
+    city: 'Paris',
+    country: 'FR'
+  },
+  members: null
+}
+export const getCodeTeam: ITeam = {
+  name: 'fakeTeamFive',
+  id: 'fakeIdTeam',
```

Comment:
fakeIdTeam -> fakeTeamId

Even though the id could reflect the name of the team (would be easier for debugging purposes)